src/components/layout/layout.js

```diff
@@ -24,32 +25,53 @@ const has_dataLayer = isBrowser() && window.dataLayer
 const TRACKING_STATUS_KEY = 'tracking_status'
 const tracking_status_cookie = new CookieStorage(TRACKING_STATUS_KEY)
 
-const cfdWarningHeight = 8
+const cfd_warning_height_desktop = 8
+const cfd_warning_height_tablet = 12
```

Comment:
Since you've declared them in utils, you can just import them instead of declaring it here again.

Reply:
@priojeet-deriv pls take note, will do this in the next PR

Reply:
nice catch. will do.